runtime/src/transaction/dispatcher.rs

```diff
@@ -59,7 +59,11 @@ pub trait Dispatcher {
         _args: cbor::Value,
     ) -> Result<cbor::Value, RuntimeError> {
         // Default implementation returns an error.
-        Err(RuntimeError::new("dispatcher", 1, "query not supported"))
+        Err(RuntimeError::new(
+            "rhp/dispatcher",
+            2,
```

Comment:
Note that the SDK uses a different error code currently for "method not found".

Reply:
Hm, but it's also a different module right, so it wouldn't work in any case? So should these common codes/runtime errors be defined somewhere in oasis-core-runtime (protocol?)?

Alternative for now would be to not specially treat "method not found" error here, which doesn't seem ideal

Reply:
Yeah I meant module+code. I think defining it in oasis-core-runtime makes sense and then reusing it in the SDK? On the other hand, does it really matter? If there is an error while fetching weights, just log it and default to no weight updates?

Reply:
Yeah i guess it doesn't really matter that much for this case, we can skip this for now, could consider it in future to have some of the error codes defined as part oh the rhp.